test.sh

```diff
@@ -74,6 +103,25 @@ openssl req -new -newkey rsa:4096 -days 365 -nodes -x509 \
 kubectl create namespace "$NAMESPACE"
 kubectl -n "$NAMESPACE" create secret tls certificates --cert=cert.crt --key=cert.key
 
+# only install prometheus helm chart for these tests
+PROMETHEUS_TESTS=(complete_values)
+for test_name in "${PROMETHEUS_TESTS[@]}"; do
```

Comment:
Looping here will install the Prometheus Operator multiple times, if we add more tests in the future. This loop seems unnecessary.

You could also simplify searching through the array with:
```
if printf '%s\0' "${TEST_NAMES[@]}" | grep -qwz complete_values
```